.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,54 @@
+name: "Step 2: Publish Release"
```

Comment:
I think this comes from the [extension template](https://github.com/jupyterlab/extension-template/tree/main/.github), as I haven't tried setting up CI/CD yet. I will do that later.

Reply:
Ok, so we can keep as is and then we improve the CI.

Reply:
I'd say that, the moment we integrate this into the mono repo of jupyter extensions, then we decide how what we do with the CI. If we can afford to redo the whole CI across all our repos, then we do it. Otherwise we just adapt this one to the same thing that is done in jupyter-extensions.

Reply:
To be discussed. I thought we wanted to keep this in a separate repo for easier contributions. So that would mean uniform CI